app/src/main/java/atlant/moviesapp/fragments/YouTubeFragment.java

```diff
@@ -0,0 +1,92 @@
+package atlant.moviesapp.fragments;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.google.android.youtube.player.YouTubeInitializationResult;
+import com.google.android.youtube.player.YouTubePlayer;
+import com.google.android.youtube.player.YouTubePlayerSupportFragment;
+
+import atlant.moviesapp.BuildConfig;
+import atlant.moviesapp.R;
+import atlant.moviesapp.presenters.YouTubePresenter;
+import atlant.moviesapp.views.YouTubeView;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class YouTubeFragment extends Fragment implements YouTubeView {
+
+    private String API_KEY = BuildConfig.YT_API_KEY;
+    private YouTubePresenter presenter;
+    private Integer movieId;
+
+    public YouTubeFragment() {
+        // Required empty public constructor
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+
+        View v = inflater.inflate(R.layout.fragment_you_tube, container, false);
+
+        movieId = getArguments().getInt("id");
+        presenter = new YouTubePresenter(this);
+        presenter.getVideo(movieId);
+
+        return v;
+    }
+
+    @Override
+    public void ShowVideo(final String videoId) {
```

Comment:
Please try to stick to the Java code writing conventions, the first letter of the method is small